versioned/persist/tx-test/src/main/java/org/projectnessie/versioned/persist/tx/postgres/PostgresTestUtil.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.tx.postgres;
+
+public final class PostgresTestUtil {
+
+  private PostgresTestUtil() {}
+
+  public static String getPostgresContainerVersion() {
+    return System.getProperty("it.nessie.container.postgres.tag", "14");
```

Comment:
Somehow I recall we agreed that a separate class isn't great, because managing dependency version in source code isn't great. Having a default version injected from the build scripts is much nicer.

Reply:
wondering:
whats the value of making the "14" externally configurable when you supposedly can already set `it.nessie.container.postgres.tag` ?

also if this is a problem, then we seem to have it in the other adapters as well:
```
versioned/persist/dynamodb-test/src/main/java/org/projectnessie/versioned/persist/dynamodb/LocalDynamoTestConnectionProviderSource.java
78:    // String version = System.getProperty("it.nessie.container.dynamodb.tag", "latest");
80:    String version = System.getProperty("it.nessie.container.dynalite.tag", "latest");

versioned/persist/mongodb-test/src/main/java/org/projectnessie/versioned/persist/mongodb/LocalMongo.java
53:    String version = System.getProperty("it.nessie.container.mongodb.tag", "4.2-bionic");

versioned/persist/tx-test/src/main/java/org/projectnessie/versioned/persist/tx/postgres/CockroachTestConnectionProviderSource.java
37:    String version = System.getProperty("it.nessie.container.cockroach.tag", "latest-v21.2");
```

Reply:
@XN137 : The places you have mentioned have only one place Initialization. 
But here, at two places we are initializing to "14". So, whenever we need to upgrade from "14" to "15" we need to change in two places. I think the issue wants us to avoid it.

I have observed some Gradle files and tried this https://github.com/projectnessie/nessie/pull/5108
If CI passes, @sagar-agarwal2404 can take that change.